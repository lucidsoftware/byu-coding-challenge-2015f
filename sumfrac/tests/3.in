10
1/10
1/3
1/40
1/6
1/60
1/24
1/7
1/64
1/5
1/2