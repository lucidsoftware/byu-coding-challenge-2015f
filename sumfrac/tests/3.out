10361/6720
